src/game/game_state.c

```diff
@@ -1150,15 +1150,12 @@ void game_state_play_sound(game_state *gs, int id, float volume, float panning,
     s.id = id;
     s.length = src_len;
     // TODO factor in pitch
-    s.duration = (int)((src_len / 8000.0f) * 1000.0f);
+    s.duration = (int)((src_len / 8000.0f) * 1000.0f); // used to track when a sound is done
```

Comment:
8000.0 * clampf(pitch, PITCH_MIN, PITCH_MAX)... maybe a helper function in the audio side to generate the expected samplerate ?